tests/CMakeLists.txt

```diff
@@ -15,6 +15,7 @@
 # along with eiquadprog.  If not, see <https://www.gnu.org/licenses/>.
 #
 
+INCLUDE_DIRECTORIES(SYSTEM ${Boost_INCLUDE_DIRS})
```

Comment:
We already have a target_link_library for each test to boost. Why would you need this ?